src/Person.java

```diff
@@ -0,0 +1,28 @@
+public class Person { public String firstName;
+    public String lastName;
+    public String city;
+    public String phone;
+    public static void personInfo(Person person){
```

Comment:
There should be three parameters for input according to the task, so the class parameters are redundant